substrate/frame/revive/Cargo.toml

```diff
@@ -65,6 +65,7 @@ pallet-revive-fixtures = { workspace = true, default-features = true }
 secp256k1 = { workspace = true, features = ["recovery"] }
 serde_json = { workspace = true }
 hex-literal = { workspace = true }
+env_logger = { workspace = true }
```

Comment:
Where is this used?

Reply:
no where I think I removed it from local tests

Reply:
ah using it in some example will move it to dev-dependencies